README.md

```diff
@@ -69,7 +69,13 @@ there are two ways to specify the host in the jolokia URL so this URL will be re
 10. If you wish to run this as a service you need to create a service wrapper for it. Any pull requests for making it are welcome! If it's possible running it as docker making it simpler.
 
    
-
+## As Java Agent
+This lib can also get the metrics from MBean Platform instead of jolokia. In order to do so, we need to run inside the JVM.
+- First, get the java agent jar from the releases page
+- Attach to your jar:  java -javaagent:jmx2graphite-1.1.0-javaagent.jar=GRAPHITE_HOSTNAME=graphite.host;SERVICE_NAME=Myservice ...
```

Comment:
"Attach to your jar" - modify your app JVM arguments and add the following ...
You didn't specify path but just wrote filename - assumption the agent jar is in the same dir. Better place it in /opt/myapp/bin just for the sake of the example